src/main/java/com/microsoft/azure/vmagent/util/Constants.java

```diff
@@ -128,7 +128,6 @@ public final class Constants {
     public static final String WINDOWS_SERVER_2022 = "Windows Server 2022";
     public static final String WINDOWS_SERVER_2019 = "Windows Server 2019";
     public static final String WINDOWS_SERVER_2016 = "Windows Server 2016";
-    public static final String UBUNTU_1604_LTS = "Ubuntu 16.04 LTS";
```

Comment:
`openjdk-17-jdk` is not available on Ubuntu 16, given its so far out of support I've removed it from the list.

Reply:
make sense to me